with-nextjs/app/api/webhooks/polar/route.ts

```diff
@@ -0,0 +1,25 @@
+import { Webhooks } from '@polar-sh/nextjs'
+import { NextResponse } from 'next/server'
+
+export const POST = async () => {
+  const webhookSecret =
+    process.env.NEXT_PUBLIC_POLAR_MODE === 'production' ? process.env.NEXT_PUBLIC_PRODUCTION_POLAR_WEBHOOK_SECRET : process.env.NEXT_PUBLIC_SANDBOX_POLAR_WEBHOOK_SECRET
```

Comment:
I like the change of the env! Just make sure that these are correctly set per my earlier comments + are in sync with .env.example